spec/features/posts_show_spec.rb

```diff
@@ -0,0 +1,17 @@
+# spec/features/post_show_spec.rb
+require 'rails_helper'
+
+RSpec.describe 'PostShow', type: :feature do
+  before do
+    @user = User.create!(name: 'User', email: 'user@example.com')
+    @post = @user.posts.create!(title: 'Sample Post', text: 'This is a sample post')
+  end
+
+  it 'shows details of a post' do
+    visit post_path(@post)
+    expect(page).to have_content(@post.title)
+    expect(page).to have_content(@post.text)
+    expect(page).to have_content('Comments: ')
+    expect(page).to have_content('Likes: ')
+  end
```

Comment:
- [ ] Just as others I have pointed, this test is to total 7 but I can see you have a chained test, I would encourage you to write the tests separately and write all the tests required for this file.
![image](https://github.com/jlberbesi/blog-app/assets/86200356/70637eee-6802-44ff-8bfe-f8f069a1fa62)

Reply:
![imagen](https://github.com/jlberbesi/blog-app/assets/88105477/d2ebf2ec-2428-453e-8656-0fda5293c65e)